.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+  "trailingComma": "es5",
+  "singleQuote": true,
+  "semi": false
```

Comment:
👎 

We're running through Babel, so let's use modern standards and options. And JavaScript has strange behaviour in some cases around semicolons, so let's not turn them off. Defaults are best.

Reply:
Cool, will do. I fully admit to not really knowing what the "best" way of doing these kind of things are, so appreciate the steer 👍